src/core/SkipToContent/SkipToContent.test.tsx

```diff
@@ -0,0 +1,85 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
Maybe rename this file (and folder) to `SkipToContentLink`. It doesn't make sense to have different names.

Reply:
They're different names because `SkipToContent` is currently made up of both `SkipToContentLink` and `SkipToContentMarker`.

Reply:
Renamed